backend/src/schema/resolvers/comments.spec.js

```diff
@@ -172,6 +175,7 @@ describe('UpdateComment', () => {
               createdAt: expect.any(String),
             },
           },
+          errors: undefined,
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-12T23:54:08Z" title="Friday, December 13th 2019, 12:54:08 am +01:00">Dec 13, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -172,6 +175,7 @@ describe('UpdateComment', () => {
               createdAt: expect.any(String),
             },
           },
+          errors: undefined,
```


Good! We should do that more often.